etc/kayobe/pulp.yml

```diff
@@ -477,6 +477,7 @@ stackhpc_pulp_images_kolla:
   - neutron-dhcp-agent
   - neutron-l3-agent
   - neutron-metadata-agent
+  - neutron-mlnx-agent
```

Comment:
This is a list of images that get synced into pulp, but doesn't control which images are built in our CI pipeline. For that you would need to set `kolla_enable_neutron_mlnx: true` in `etc/kayobe/environments/ci-builder/stackhpc-ci.yml`.

For context, this adds the regex to the kolla-build command https://opendev.org/openstack/kayobe/src/branch/master/ansible/inventory/group_vars/all/kolla#L206